### PR TITLE
Fix struct names

### PR DESCRIPTION
**Pull request by coderwander**

### CODE REVIEW

**Review by guggero — APPROVED** [review]

**Comment by guggero** [comment]
Yes, exercise patience and wait for a second reviewer to approve.

**Review by jcvernaleo — APPROVED** [review]
OK

**Comment by jcvernaleo** [comment]
Sorry for the delay there.  I'm more than a little behind :)

**Pull request merged**
